def/babel6.js

```diff
@@ -0,0 +1,117 @@
+module.exports = function (fork) {
+    fork.use(require("./babel"));
+    fork.use(require("./flow"));
+
+    // var types = fork.types;
+    var types = fork.use(require("../lib/types"));
+    // var defaults = fork.shared.defaults;
+    var defaults = fork.use(require("../lib/shared")).defaults;
+    var def = types.Type.def;
+    var or = types.Type.or;
+
+    def("Directive")
+        .build("value")
+        .field("value", def("DirectiveLiteral"));
+
+    def("DirectiveLiteral")
+        .build("value")
+        .field("value", String, defaults["use strict"]);
+
+    def("BlockStatement")
+        .bases("Statement")
+        .build("body")
+        .field("body", [def("Statement")])
+        .field("directives", [def("Directive")]); // add directive
```

Comment:
I would suggest that you try to make the builders defined for Babel compatibility as similar to the `babel-types` builders as possible. Trying to context switch between `babel-types` / `ast-types` builders is already enough to drive you crazy, no need make `ast-types/babel` some weird mixture of the two and further confuse things.